Source/WebCore/platform/graphics/cairo/ImageBufferUtilitiesCairo.cpp

```diff
@@ -78,31 +78,11 @@ static bool encodeImage(cairo_surface_t* surface, const String& mimeType, std::o
     // List of supported image encoding types comes from the GdkPixbuf documentation.
     // http://developer.gnome.org/gdk-pixbuf/stable/gdk-pixbuf-File-saving.html#gdk-pixbuf-save-to-bufferv
 
-    String type = mimeType.substring(sizeof "image");
+    String type = imageTypeFromMIMEType(mimeType);
```

Comment:
This change broke the GTK+ build. There is no `imageTypeFromMIMEType()` defined anywhere, but I see it mentioned in the commit log. Did you forget to add the chunk to the commit?